src/attributes.js

```diff
@@ -93,6 +93,21 @@ class Attributes {
         return this;
     };
 
+    // Class names of variables whose names contain 'attributes' are merged
+    // instead of being replaced.
+    // @see TwigAdapter:Twig.Template.prototype.render()
+    // @see TwigAdapter/tags:render()
```

Comment:
1. Should use JSDoc block syntax instead of inline comments.

2. Static helper functions should be declared somewhere else than the middle of all other functions; either at the very beginning or at the very end of the class.

Reply:
You're right. Since it was called `convertAttributes()` I wanted to stay it close to the other `<x>Attribute()` methods. Since I changed the name while `Attribute` became obsolete it makes more sense to place it somewhere else. Thanks for pointing out.